packages/vscode-extension/src/telemetry/extTelemetryEvents.ts

```diff
@@ -36,6 +36,7 @@ export enum TelemetryEvent {
 
   F5Start = "f5-start",
   F5 = "f5",
+  StopDebug = "stop-debug",
```

Comment:
What's the difference of "f5" and "stop-debug" events? I think we should send "f5" at debug session termination. No need to add an event called "stop-debug".

Reply:
Actually we need three events for debug. I've changed the name to "debug-precheck", "debug-start", and "debug-stop"